### ISSUE TITLE
Reach GPU RAM limit in 10 image generated

### ISSUE THREAD

**Issue by Semonxue** [camenduru/Fooocus-colab#2]
Thanks Gamenduru and lllyasviel , Fooocus is good for easily generating midjourney quality images. 

After I tested , I found the app will crash after about 10 images in colab. It will touch GPU ram limit very soon.

I'm run on colab pro and T4 server.

Please check the screenshot when crash in below.
![Screenshot of Fooocus_colab ipynb - Colaboratory](https://github.com/camenduru/Fooocus-colab/assets/372518/ec186956-0718-45be-af7b-18c4d5d021a5)

**Comment by kuldp18** [comment]
> Thanks Gamenduru and lllyasviel , Fooocus is good for easily generating midjourney quality images.
> 
> After I tested , I found the app will crash after about 10 images in colab. It will touch GPU ram limit very soon.
> 
> I'm run on colab pro and T4 server.
> 
> Please check the screenshot when crash in below. ![Screenshot of Fooocus_colab ipynb - Colaboratory](https://user-images.githubusercontent.com/372518/260237489-ec186956-0718-45be-af7b-18c4d5d021a5.jpg)

Also the --normalvram flag doesn't help :(

**Comment by camenduru** [comment]
yep, we have a memory leak 😱 maybe it is in the comfyui code, idk yet 😭

**Comment by kuldp18** [comment]
> yep, we have a memory leak 😱 maybe it is in the comfyui code, idk yet 😭

so I should avoid running it locally as well right?

**Comment by camenduru** [comment]
![image](https://github.com/camenduru/Fooocus-colab/assets/54370274/8e3e31e5-0133-42ab-953b-772ee8b5bf0e)

maybe fixed please try again

**Comment by Semonxue** [comment]
@camenduru Thanks for hard working. It looks the problem is still there. I tried 2 times and still crashed in 20 images.

![image](https://github.com/camenduru/Fooocus-colab/assets/372518/72918350-090e-4416-87a8-b539c4d087c7)
![image](https://github.com/camenduru/Fooocus-colab/assets/372518/93aa752f-b745-4866-b0f7-e4942324b389)
